src/yunohost/log.py

```diff
@@ -289,6 +290,31 @@ def func_wrapper(*args, **kwargs):
     return decorate
 
 
+class RedactingFormatter(Formatter):
+
+    def __init__(self, format_string, data_to_redact):
+        super(RedactingFormatter, self).__init__(format_string)
+        self.data_to_redact = data_to_redact
+
+    def format(self, record):
+        msg = super(RedactingFormatter, self).format(record)
+        self.identify_data_to_redact(msg)
+        for data in self.data_to_redact:
+            msg = msg.replace(data, "**********")
+        return msg
+
+    def identify_data_to_redact(self, record):
+
+        # Wrapping this in a try/except because we don't want this to
+        # break everything in case it fails miserably for some reason :s
+        try:
+            match = re.search(r'(db_pwd|password)=(\S{3,})$', record.strip())
```

Comment:
What about adding a more generic thing to be able to support all variable containg `pwd`.

Maybe we could also try to add some other key word like `pass` or `token`.